src/service/xhr-impl.js

```diff
@@ -223,7 +223,7 @@ function setupJson_(init) {
   init.headers = init.headers || {};
   init.headers['Accept'] = 'application/json';
 
-  if (init.method == 'POST') {
+  if (init.method == 'POST' && init.type == 'json') {
```

Comment:
What is `init.type`? Is it part of the spec? I've never seen it. Could you please send me a reference to the doc?